sqjobs/contrib/django/djsqjobs/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+import pytz
+
+
+class JobStatus(models.Model):
+    PENDING = 'PENDING'
+    SUCCESS = 'SUCCESS'
+    FAILURE = 'FAILURE'
+    TIMEOUT = 'TIMEOUT'
+    states = (
+        (PENDING, 'PENDING'),
+        (SUCCESS, 'SUCCESS'),
+        (TIMEOUT, 'TIMEOUT'),
+        (FAILURE, 'FAILURE')
+    )
+
+    """Job result/status."""
+    job_id = models.CharField(u"job id", max_length=255, unique=True)
+    job_name = models.CharField(u"job name", max_length=255, unique=False)
+    status = models.CharField(
+        u"job status", max_length=7, default=PENDING, choices=states
+    )
+    result = models.TextField(null=True, default=None)
+    date_started = models.DateTimeField(u"started at", auto_now_add=True)
+    date_done = models.DateTimeField(u"done at", auto_now=False, null=True)
+    traceback = models.TextField(u"traceback", blank=True, null=True)
+
+    def to_dict(self):
+        return {"job_id": self.job_id,
+                "job_name": self.job_name,
+                "status": self.status,
+                "result": self.result,
+                "date_started": self.date_done,
+                "date_done": self.date_done,
+                "traceback": self.traceback}
+
+    def __unicode__(self):
+        return u"<Job (%s): %s state->%s>" % (self.job_name, self.job_id, self.status)
```

Comment:
state here maps to status. When there are three parameters like in here sometimes is better to use format:

```
return "<Job ({name}): {id} status->{status}>".format(
    name=self.job_name,
    ...
)
```